doc/api/deprecations.md

```diff
@@ -3360,6 +3360,21 @@ Type: Runtime
 In a future version of Node.js, the `asyncResource` property will no longer
 be added when a function is bound to an `AsyncResource`.
 
+### DEP0173: `util.toUSVString`
+
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/47342
+    description: Documentation-only deprecation.
+-->
+
+Type: Documentation-only
+
+[`util.toUSVString()`][] is replaced with [`String.prototype.toWellFormed`][]
+after V8 11.3. It is recommended to use the built-in function for performance
+reasons.
```

Comment:
This should mention that now it is just an alias.

Reply:
Thanks for the review Matteo. I just updated the description.